prime-router/settings/organizations-local.yml

```diff
@@ -405,6 +429,7 @@
           useBatchHeaders: true
           receivingApplicationName: NEDSS
           receivingFacilityName: TX-ELR
+          defaultAoeToUnknown: true
```

Comment:
Texas wants to have ALL the AOE questions come through with `UNK` if they don't have an answer, so every message in the HL7 doc will have all the AOE questions, even if they're not asked or transmitted.